src/main/java/com/ays/assignment/model/entity/AssignmentEntity.java

```diff
@@ -67,9 +69,10 @@ public class AssignmentEntity extends BaseEntity {
     private UserEntity user;
 
     public void setPoint(double latitude, double longitude) {
-        Coordinate coordinate = new Coordinate(latitude, longitude);
+        Coordinate[] coordinates = new Coordinate[]{new Coordinate(latitude, longitude)};
+        CoordinateSequence coordinateSequence = new CoordinateArraySequence(coordinates);
         GeometryFactory geometryFactory = new GeometryFactory();
-        this.point = geometryFactory.createPoint(coordinate);
+        this.point = geometryFactory.createPoint(coordinateSequence);
     }
```

Comment:
Buradaki kodu Postman üzerinden istek atarak test ettik mi? Veritabanına kayıt ediliyor mu?

Reply:
Assignment Institution olayı One-to-One yaptığımızdan aynı institution id'ye farklı assignment ekleyemiyoruz.
Liquibase tarafı düzenledim.

One-to-One konuşmamız lazım.

Onun dışında sonuç şu şekilde. 

@agitrubard 

![afet_assignment_save](https://github.com/afet-yonetim-sistemi/ays-be/assets/19342017/088a7c41-c2bf-4008-bb94-be6694215cbe)